manifests/r/ramdor/Thetis/2.10.3.8/ramdor.Thetis.locale.en-US.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with komac v2.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: ramdor.Thetis
+PackageVersion: 2.10.3.8
+PackageLocale: en-US
+Publisher: Richie - MW0LGE
+PublisherUrl: https://github.com/ramdor
+PublisherSupportUrl: https://github.com/ramdor/Thetis/issues
+Author: Richie - MW0LGE
+PackageName: Thetis
+PackageUrl: https://github.com/ramdor/Thetis
+License: GPL-2.0
+LicenseUrl: https://github.com/ramdor/Thetis/blob/master/LICENSE
+ShortDescription: The main working repo for changes to Thetis for the Apache Labs line of radios.
```

Comment:
@Dragon1573 This is not a correct description. There is no (Git) "repo" in the manifest, just the app itself. The proper description should be:

`Client software for HPSDR with changes for the Apache Labs line of radios.`